### PR TITLE
Cem 2572

### PR DESCRIPTION
**Pull request by brandontiu**
Branched off master and added changes to HighlightedText.tsx

### CODE REVIEW

File: src/Containers/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -22,6 +22,8 @@ export interface IDropdownProps extends React.HTMLAttributes<HTMLDivElement> {
     dropdownButton: React.ReactElement;
     dropdownWidth?: number;
     right?: boolean;
+    startState?: boolean;
```
Comment (ralph-dev) [inline src/Containers/Dropdown/Dropdown.tsx:25]: Why is this here?

**Review by ralph-dev — CHANGES_REQUESTED** [review]
Did you get this from master?

File: src/Containers/Dropdown/Dropdown.tsx (hunk 1 above)
Comment (brandontiu) [reply src/Containers/Dropdown/Dropdown.tsx]: My bad, I didn't update my master branch before I cloned it. It should be fixed now.

File: src/Containers/HighlightedText/HighlightedText.tsx [hunk 2]
```diff
@@ -103,3 +110,8 @@ export const HighlightedText: React.FC<HighlightedTextProps> = ({
 };
 
 const HighlightedRow = styled.div``;
+
+const StyledClickableSmallText = styled(ClickableSmallText)`
+    color: ${({bold}) => bold ? 'orange' : 'black'}
```
Comment (ralph-dev) [inline src/Containers/HighlightedText/HighlightedText.tsx:115]: Color should come from theme

**Review by ralph-dev — APPROVED** [review]
Lgtm needs to be fixed and I've ignored some small issues

File: src/Containers/HighlightedText/HighlightedText.tsx [hunk 3]
```diff
@@ -103,3 +111,7 @@ export const HighlightedText: React.FC<HighlightedTextProps> = ({
 };
 
 const HighlightedRow = styled.div``;
+
+const StyledClickableSmallText = styled(ClickableSmallText)`
+    color: ${({bold}) => bold ? MainTheme.colors.statusColors.orange : 'black'}
```
Comment (ralph-dev) [inline src/Containers/HighlightedText/HighlightedText.tsx:116]: Use theme for black

**Review by ralph-dev — APPROVED** [review]
Lgtm

**Review by ralph-dev — APPROVED** [review]
Lgtm
